src/ucs/datastruct/dynamic_bitmap.h

```diff
@@ -99,13 +99,27 @@ ucs_dynamic_bitmap_reserve(ucs_dynamic_bitmap_t *bitmap, size_t num_bits)
 }
 
 
+/**
+ * Get the number of bits in the bitmap.
+ *
+ * @param [in] bitmap  Find the length of this bitmap.
```

Comment:
```suggestion
 * @param [in] bitmap  Find the number of bits set in this bitmap.
```
If I understand correctly, length and number of bits aren't always the same. Are they?